### ISSUE TITLE
[A11y][Link][Android] Link role is missing

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#448]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

S22

### OS Version

16

### App version

0.6.0 alpha (441)

### Describe the issue

Link alone is read "Label" by Talkback : role "link" is missing
Link + icon is read "Label, **button**" : role "link" should be read
Link + chevron is read "Label, back/next image": role "link" is missing and chevron should be hidden

### Expected Behavior

see above

**Comment by nouha06** [comment]
The issue has been fixed and the ticket has been closed, as we cannot release a new version. That’s why the TestFlight blocking label was applied.
